src/firecracker/swagger/firecracker.yaml

```diff
@@ -854,6 +854,12 @@ definitions:
       reg_modifiers:
         type: object
         description: A collection of registers to be modified. (aarch64)
+      vcpu_features:
```

Comment:
LGTM, I would mention the fix in the changelog

Reply:
ack